localization/languages/hu.json

```diff
@@ -102,10 +102,10 @@
         "settingsSwipeNavigationHeading": "Swipe Navigáció",
         "settingsSwipeNavigationToggle": "Kapcsolja be a swipe Navigációt előre és hátra az oldalakal.",
         "settingsSearchEngineHeading": "Kereső Motor",
-        "settingsDefaultSearchEngine": "Válasza ki Alap kereső Motorját:",
-        "settingsDDGExplanation": "Set DuckDuckGo as the default search engine to see instant answers in the searchbar.",
+        "settingsDefaultSearchEngine": "Válasza ki az Alap kereső Motorját:",
+        "settingsDDGExplanation": "Állitsa be a DuckDuckGo kereső motort hogy lássa a válaszokat a kereső sorba.",
         "settingsKeyboardShortcutsHeading": "Billentjüzet kombinációk",
-        "settingsKeyboardShortcutsHelp": null, //missing translation
+        "settingsKeyboardShortcutsHelp": "Irja a \"mod\" hogy használja a command vagy kontroll gombot.", //missing translation
```

Comment:
Starting in 1.7.0, this string has actually been changed to "use commas to separate multiple shortcuts", which is why I removed the previous translation. Would you mind translating the new string?